fcrepo-clients/src/main/java/edu/unc/lib/dl/fcrepo4/Repository.java

```diff
@@ -80,6 +82,18 @@
 
     private RepositoryObjectDataLoader repositoryObjectDataLoader;
 
+    private LoadingCache<PID, RepositoryObject> contentObjCache;
```

Comment:
maybe this field should be named "repositoryObjCache" to make it clearer what's being cached?